doc/README.txt

```diff
@@ -1,11 +1,15 @@
 This directory supports the creation of the Gcovr User Guide using
-docutils (`rst2html5.py`).
+sphinx.
+
+The following python packages have to be installed, e.g. with pip:
+
+    pip install sphinx sphinx_rtd_theme
```

Comment:
Or should this be executed in a virtualenv and requirements.txt should be provided?

Reply:
Currently all development prerequisites are documented in the CONTRIBUTING document. You can mention the sphinx dependencies there. If you list the deps in a requirements.txt that would probably be even better (in which case CONTRIBUTING and doc/README and possibly also .travis.yml and appveyor.yml can be simplified).

Reply:
I added a requirements.txt without pinning the versions.